anomalib/models/__init__.py

```diff
@@ -24,14 +26,40 @@
 from anomalib.models.components import AnomalyModule
 
 
+def _snake_to_camel_case(model_name: str) -> str:
```

Comment:
I could be wrong, but I think the style we use here is PascalCase (in camelCase, the first word starts with a small leter).

Reply:
yeah, good catch. That should be pascal case.